bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/dialogs/NewWizardNewPage.java

```diff
@@ -689,4 +689,11 @@ public IWorkbenchWizard createWizard() throws CoreException {
 
 		updateDescription(selectedObject);
 	}
+
+	/**
+	 * @return Returns the selectedElement.
```

Comment:
Comment is meaningless. Instead, one could provide information that the selection could be null